packs/travisci/actions/list_repos.py

```diff
@@ -0,0 +1,19 @@
+from lib.action import TravisCI
+import requests
+import yaml
+
+
+class GetRepos(TravisCI):
+    def run(self, username):
+        """
+        Listing all Repos for a give user
+        """
+        _HEADERS = self.travis
+        _HEADERS['Authorization'] = self.config["Authorization"]
+        uri = self.config["uri"] + '/repos?owner_name=' + username
+        response = requests.get(uri, headers=_HEADERS)
+        data = yaml.load(response.content)
```

Comment:
I'm slightly confused, we send "Accept: application/vnd.travis-ci.2+json" header, but you are using YAML to parse the response here?